config-dev/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
What about a layout like this?

```
config/
├── base/
│   ├── configmap.yaml
│   ├── event-source.yaml
│   ├── knative-service.yaml
│   ├── rbac.yaml
│   ├── trigger.yaml
│   └── kustomization.yaml
├── dev/
│   ├── kustomization.yaml
```

`config/dev/kustomization.yaml` would contain 

```yaml
resources:
  - ../base
   # Tekton
  - https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
  # Konflux Snapshot CRD
  - https://raw.githubusercontent.com/konflux-ci/application-api/refs/heads/main/manifests/application-api-customresourcedefinitions.yaml
   ```
   
   Install would change to `kustomize build config/dev/ | ko apply -f -`

Reply:
Sounds good to me.